ovos_utils/sound/__init__.py

```diff
@@ -2,22 +2,57 @@
 import time
 from ovos_utils.log import LOG
 from ovos_utils.signal import check_for_signal
+from distutils.spawn import find_executable
 
 
-def play_audio(uri, play_cmd="play %1"):
+def play_audio(uri, play_cmd=None):
     """ Play a audio file.
 
         Returns: subprocess.Popen object
     """
-    play_wav_cmd = str(play_cmd).split(" ")
-    for index, cmd in enumerate(play_wav_cmd):
+    sox_play = find_executable("play")
+    pulse_play = find_executable("paplay")
+    alsa_play = find_executable("aplay")
+    mpg123_play = find_executable("mpg123")
+
+    player = play_cmd
+
+    # NOTE: some urls like youtube streams will cause extension detection to fail
+    # let's handle it explicitly
+    ext = uri.split("?")[0].split(".")[-1]
+    # Replace file:// uri's with normal paths
+    uri = uri.replace('file://', '')
+
+    # sox should handle almost every format, but fails in some urls
+    if sox_play:
```

Comment:
Would it be slightly more efficient to just append the URI here instead of `%1`?